app/views/desks/index.html.erb

```diff
@@ -1,7 +1,22 @@
-<h3>Desk list</h3>
+<div class="alert alert-warning alert-dismissible fade show" role="alert">
+Click on the status to see all desk details!
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
+
+<div class="row desk-row">
 <% @desks.each do |desk| %>
-  <ul>
-    <li><%= link_to desk.name, desk_path(desk) %></li>
-    <li id=<%="desk_state_#{desk.id}" %>><%= desk.occupied %></li>
-  </ul>
+    <div class="col-6-auto desk-card-col">
+      <div class="card desk-card">
+        <h5 class="card-header"><%= desk.name %></h5>
+        <div class="card-body">
+          <p class="card-title">Desk ID: <%= desk.id %></p>
+          <div class="text-center">
+            <%= link_to desk_status(desk), desk_path(desk), class:"btn btn-outline-#{desk_class(desk)}", id: desk_element_id(desk)%>
```

Comment:
In my opinion, the use of helpers here makes this less readable/localized. Instead of having all the information in one place we have to go to helpers.
Especially since all the helpers are 1 liners so it would be easy to do them inline.

I see they are used elsewhere but I still kind of think it's unnecessary. What do you think?

On a separate note, add a space after `class:`

Reply:
I think that when we expand our UI we will rely on those helpers more, and if/when we need to change the class type or the display text of the element, it would be easy to change it in one place (the helper) rather than each occurrence in the code.

Reply:
I strongly believe that there should be the least amount of coupling between the logic and the view, as in they shouldn't care about how you determine the class depending on an object's attribute.

It may not look like much because our logic is pretty simple, but as our logic gets more complex our view will get a lot more cluttered and in some cases we'll repeat ourselves. Also, it allows us to test said logic which imo is pretty essential which we wouldn't be able to do if the logic is embedded in the view.